rfc/guides-comments.md

```diff
@@ -0,0 +1,75 @@
+Options available for adding comments to the deployment guides are as follows:
+
+1.  Disqus: <https://disqus.com/>
+
+Really good third party service and the most popular. It has 3 options:
+
+-   Basic: This has ad's (not recommended)
+
+-   Plus $9 per month (Suitable for under 50k visitors daily)
+
+-   Pro $89 per month (Suitable for 150k visitors daily)
+
+Pros:
+
+-   It provides a dashboard to manage for all the comments posted.
+
+-   Analytics available for comments on the dashboard.
+
+-   Very easy to install
+
+Cons
+
+-   Being a third party service, we don't know what our comments become and how disqus uses the data retrieved from the comments 
+
+-   Disqus doesn't have a great track record on privacy
+
+-   We cannot control the look and feel of the design
+
+-   Does not allow anonymous content
+
+1.  Just comments  [https://just-comments.com/](https://just-comments.com/pricing.html)
+
+Also a third party service and very similar to Disqus
+
+Pros:
+
+-   Look and feel of the design is customizable
+
+-   Pay as you go but you will need to pay 5.30 dollars for having an active api key per month
+
+Cons
+
+-   We have no control over the data
+
+1.  Commento  <https://commento.io/>
+
+Pros
+
+-   The data retrieved is ours
+
+-   Pay what we want <https://commento.io/pricing>
+
+Cons
+
+-   We have to host the solution
+
+1.  Staticman:  <https://staticman.net/>
+
+Pros:
+
+-   Designed to work with jekyll
+
+-   Complete control over the data/content, user experience, and user interface.
+
+-   Not just for comments! Perfect for any sort of user generated content: reviews, comments, polls, and more.
+
+-   Allows comments to be posted without any login required.
+
+Cons:
+
+-   Not sure if it is a downside but Staticman access to your Jekyll repository
```

Comment:
Intriguing! What a neat concept. I do have slight concern about this, and also the potential for some sort of injection attack if we're not careful about how we read & render the comment files. But overall, this is quite promising!